### ISSUE TITLE
Number of characters (per note)

### ISSUE THREAD

**Issue by GEkuL** [kmac/mindfulnotifier#25]
Could you increase the number of characters per note. 256 for me is not enough, could it be upgraded for 1024?)

**Comment by kmac** [comment]
Hmm, I'm not sure if the notification area would support that. Right now the limit is just a setting at the UI level, so if you want to try it you could do an export, then manually edit the entry in the JSON file. Then re-import. Do you mind trying that? Just to see if the notification can handle it.

**Comment by GEkuL** [comment]
@kmac yaaah, the problem is that I tried it with import, and some messages are shown cropped, and some aren't shown at all((

**Comment by kmac** [comment]
Hmm okay. I'd have to try it and see what's happening then. I did choose the 256 char limit to be in line with a short reminder on purpose though... Sort of like an SMS or twitter. I wonder if adding some markdown like in your Issue #23 might be really what you're looking for - assuming I could get hyperlinks working - clicking on the link would take you to somewhere you have more detailed notes captured. Just a thought.

BTW I'm starting a new job this week. I don't expect to have a huge amount of time over the next month or two to add new stuff...

**Comment by wolftune** [comment]
Mentioned in closed duplicate issue:

An alternative that might be easier: make it so that tapping on the remind (to see the whole thing within the app) will count for clearing the reminder.

It's tedious to tap to have to read the whole thing but then still have to manually clear the notification

**Comment by kmac** [comment]
@wolftune This is already supported. Uncheck preferences => Use sticky notification. It will then dismiss the notification when you tap on it. I have it set to sticky notifications on by default just as a personal preference.

**Comment by kmac** [comment]
Also I will likely have to truncate the long reminder at some length for the notification window. I'm not sure what the max is, but I don't think it should be a super long field in the notification itself... I think clicking on the notification and bringing you to the main screen where it can scroll should suffice, no?

**Comment by kmac** [comment]
Fixed in 1.0.15. I have set the max in the UI to 4096 chars. My only concern with large reminders is that this stuff isn't stored in a database, it's just using the android shared preferences as of now; I'm not sure how that will handle things as the number of reminders grows and the text is large... 

* Reminders get truncated in the notification and in the list of reminders.
* Scrolling is enabled in the main display in the app. I made the font a little smaller at a certain character threshold
